codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/customizations/ServerRequiredCustomizations.kt

```diff
@@ -51,11 +51,19 @@ class ServerRequiredCustomizations : ServerCodegenDecorator {
         rustCrate.mergeFeature(
             Feature(
                 "aws-lambda",
-                true,
+                false,
```

Comment:
Why this change?

Reply:
I changed the default feature flag to `false` so that if the generated SDK is not intended for AWS Lambda, it won't automatically enable the `aws-lambda` feature on the `aws-smithy-http-server` crate.